packages/liquidator/src/liquidator.js

```diff
@@ -239,7 +253,7 @@ class Liquidator {
         message: "Detected a liquidatable position",
         scaledPrice: scaledPrice.toString(),
         maxCollateralPerToken: maxCollateralPerToken.toString(),
-        position: JSON.stringify(position)
```

Comment:
these `JSON.strigify` parts were removed as the logs look way better as structured objects spread out by the transport.